package.json

```diff
@@ -1,33 +1,36 @@
 {
-  "name": "wolfsbane-backend",
-  "version": "1.0.0",
-  "description": "A travel booking server",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start:dev": "cross-env NODE_ENV=development nodemon --exec babel-node ./src/index.js"
-  },
-  "author": "Andela Simulations Programme",
-  "license": "MIT",
-  "dependencies": {
-    "body-parser": "^1.18.3",
-    "cors": "^2.8.4",
-    "dotenv": "^6.0.0",
-    "errorhandler": "^1.5.0",
-    "express": "^4.16.3",
-    "jsonwebtoken": "^8.3.0",
-    "passport": "^0.4.0"
-  },
-  "devDependencies": {
-    "@babel/cli": "^7.5.5",
-    "@babel/core": "^7.5.5",
-    "@babel/node": "^7.5.5",
-    "@babel/preset-env": "^7.5.5",
-    "@babel/register": "^7.5.5",
-    "cross-env": "^5.2.0",
-    "eslint": "^5.16.0",
-    "eslint-config-airbnb-base": "^13.1.0",
-    "eslint-plugin-import": "^2.18.2",
-    "nodemon": "^1.19.1"
-  }
+	"name": "wolfsbane-backend",
+	"version": "1.0.0",
+	"description": "A travel booking server",
+	"main": "index.js",
+	"scripts": {
+		"test": "cross-env NODE_ENV=test && mocha src/test --require @babel/register --recursive --exit || true",
+		"start:dev": "cross-env NODE_ENV=development nodemon --exec babel-node ./src/index.js"
+	},
+	"author": "Andela Simulations Programme",
+	"license": "MIT",
+	"dependencies": {
+		"body-parser": "^1.18.3",
+		"chai": "^4.2.0",
+		"chai-http": "^4.3.0",
```

Comment:
This should be in `devDependencies`

Reply:
Thanks, @eokwukwe , Those requested changes have been done